src/main/java/net/minecraftforge/common/extensions/IForgeItem.java

```diff
@@ -533,6 +535,33 @@ default boolean canApplyAtEnchantingTable(ItemStack stack, Enchantment enchantme
         return enchantment.category.canEnchant(stack.getItem());
     }
 
+    /**
+     * Gets the level of the enchantment currently present on the stack. By default, returns the enchantment level present in NBT.
+     * Most enchantment implementations rely upon this method.
+     * For consistency, results of this method should be the same as getting the enchantment from {@link #getAllEnchantments(ItemStack)}
```

Comment:
I wonder if it is worth mentioning something along the lines of the difference between these two methods and the ones left in EnchantmentHelper so people can have a better idea of what cases they may care about NBT and ignore non NBT ones.

Reply:
Not on this doc, but maybe on `IForgeItemStack`. I treated the docs on `IForgeItem` as info for the modder overriding the method, while the docs on `IForgeItemStack` were for modders wanting enchantments.